manual/source/topic/arena.rst

```diff
@@ -159,6 +159,17 @@ Client arenas
       may pause the :term:`client program` for. See
       :c:func:`mps_arena_pause_time_set` for details.
 
+    * :c:macro:`MPS_KEY_ARENA_EXTENDED` (type :c:type:`mps_fun_t`) is
+      a function that will be called when the arena is *extended*:
```

Comment:
m. Rather then "when" say "immediately after". rule.generic.clear

Reply:
Changed to "immediately before" and "immediately after" as appropriate, with some other changes to words in the same paragraphs to conform